cli/src/index.ts

```diff
@@ -158,7 +158,12 @@ const options = {
         describe: "Skip contract verification",
         type: "boolean",
         default: false,
-    }
+    },
+    payerPath: {
+        describe: "Path to the payer json file",
+        default: false,
```

Comment:
this is a little weird here, because we specify the type to be a string, then provide a boolean as a default value, which is not type correct. instead, just don't provide a default value, in that case the type of `argv["payer"]` will be `string | undefined`